File: beginner/c/1065.c
#include <stdio.h>
int main(){

    int i, n, cont = 0;

    for(i = 0; i < 5; i++){
        scanf("%d", &n);
        if(n % 2 == 0){
            cont+=1;
        }
    }

    printf("%d valores pares\n", cont);

    return 0;
    
}

File: beginner/java/1065.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Locale.setDefault(Locale.US);

        Scanner scanner = new Scanner(System.in);

        int i, n, cont = 0;

        for(i = 0; i < 5; i++){
            n = scanner.nextInt();
            if(n % 2 == 0){
                cont+=1;
            }
        }

        System.out.println(cont + " valores pares");

    }
}

